### PR TITLE
adding automation scripts for MCSC-208 and MCSC-242

### PR DESCRIPTION
**Pull request by lahiruwijesingheOC**
Please review and approve.

### CODE REVIEW

**Comment by dacohm** [comment]
Please update the README with the correct sm2 profile information.

File: src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala [hunk 1]
```diff
@@ -141,7 +141,140 @@ class UserJourneyTests extends BaseSpec {
...
+
+      Then("I Should see the when-stop-paying-public-pension page")
+      WhenStopPayingPublicPensionPage.onWhenStopPayingPublicPensionPage()
+      //*click continue
+      When("I click continue button")
+      WhenStopPayingPublicPensionPage.submitPage()
+      //*Verify error
+      Then("I Should see the error message")
+      WhenStopPayingPublicPensionPage.validateNoInputDateError()
+      //*select 'Yes and continue'
```
Comment (dacohm) [inline src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala:235]: Remove comments.

File: src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala [hunk 2]
```diff
@@ -141,7 +141,140 @@ class UserJourneyTests extends BaseSpec {
...
+
+      Then("I Should see the error message")
+      WhenStopPayingPublicPensionPage.validatePreRemedyDateError()
+
+      When("I enter a valid last past date")
+      WhenStopPayingPublicPensionPage.enterValidPastDateAndClickContinue()
+
+      /*Then("I Should see the ###### page")
+      Then("I Should see the ###### page")
+      ###########.onWhenStopPayingPublicPensionPage().onWhenStopPayingPublicPensionPage()*/
```
Comment (dacohm) [inline src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala:260]: Remove comments.

File: src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala [hunk 3]
```diff
@@ -141,7 +141,140 @@ class UserJourneyTests extends BaseSpec {
...
+      When("I enter a valid future date")
+      WhenStopPayingPublicPensionPage.enterAFutureDateAndClickContinue()
+
+      /*Then("I Should see the ###### page")
+      ###########.onWhenStopPayingPublicPensionPage()*/
+
+      //when-stop-paying-public-pension - Verify new page url,header,title
+      //*click continue
+      //*Verify error
+      //*select 'Yes and continue'
```
Comment (dacohm) [inline src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala:277]: Remove comments.

**Review by dacohm — CHANGES_REQUESTED** [review]

File: src/test/scala/uk/gov/hmrc/test/ui/constants/Errors.scala [hunk 4]
```diff
@@ -17,13 +17,27 @@
...
+  val ERROR_SUMMARY_TITLE                                                = "There is a problem"
+  val RADIO_BUTTON_ERROR_SUMMARY                                         = "You must select one of the options"
+  val TEXT_AREA_ERROR_SUMMARY                                            = "Enter your reason"
+  val CHECK_BOX_ERROR_SUMMARY                                            = "You must select at least one of the options"
+  val TEXT_WITH_500_CHARACTERS                                           =
     "Arsenal manager Mikel Arteta also paid tribute to the Arsenal women's team, who have sold out the Emirates Stadium for tonight's Champions League semi-final second leg against Wolfsburg.The women's team, which won the Champions League in [20], lifted the League Cup earlier this season.It shows the unity, said Arteta. It shows as well the way everything is evolving together and also the importance of women's football in this country, especially wit what the England national team are doing.Testsme"
-  val TEXT_WITH_503_CHARACTERS                           =
+  val TEXT_WITH_503_CHARACTERS                                           =
```
[the remark this answers is not in the document]
Comment (lahiruwijesingheOC) [reply src/test/scala/uk/gov/hmrc/test/ui/constants/Errors.scala]: done

File: src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala [hunk 5]
```diff
@@ -141,7 +141,140 @@ class UserJourneyTests extends BaseSpec {
...
+      When("I click continue button")
+      WhichYearsScottishTaxpayer.submitPage()
+
+      Then("I Should see the error message")
+      WhichYearsScottishTaxpayer.validateWhichYearsScottishTaxpayerErrorsWhenNoCheckBoxSelected()
+
+      //which-years-scottish-taxpayer - Verify new page url,header,title
+      //*click continue
+      //*Verify error
+      //*select 'Yes and continue'
```
[the remark this answers is not in the document]
Comment (lahiruwijesingheOC) [reply src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala]: done

File: src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala (hunk 1 above)
Comment (lahiruwijesingheOC) [reply src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala]: done

File: src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala (hunk 3 above)
Comment (lahiruwijesingheOC) [reply src/test/scala/uk/gov/hmrc/test/ui/specs/UserJourneyTests.scala]: done

**Review by dacohm — APPROVED** [review]

**Pull request merged**